src/screens/FeedScreen/Content.js

```diff
@@ -336,6 +331,12 @@ export const styles = StyleSheet.create({
     justifyContent: 'space-between',
     marginLeft: 13
   },
+
+  containerFeedText: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    marginTop: 5
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New styles have been added to the `containerFeedText` object. As long as these styles align with the intended design, this change should be fine.
```diff
+  containerFeedText: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    marginTop: 5
+  },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->